eng/Generation.psm1

```diff
@@ -85,7 +85,7 @@ function Invoke-TypeSpec($baseOutput, $projectName, $mainFile, $arguments="", $s
         $repoRootPath = Join-Path $PSScriptRoot ".."
         $repoRootPath = Resolve-Path -Path $repoRootPath
         Push-Location $repoRootPath
-        $autorestCsharpBinPath = Join-Path $repoRootPath "artifacts/bin/AutoRest.CSharp/Debug/net7.0/AutoRest.CSharp.dll"
+        $autorestCsharpBinPath = Join-Path $repoRootPath "artifacts/bin/AutoRest.CSharp/Debug/net8.0/AutoRest.CSharp.dll"
```

Comment:
Why are we hardcoding these paths everywhere? That is generally not a good idea and as you can see makes updates a lot more difficult.

Reply:
yeah this is not a good idea to hard code this. We might need to change this.
Could you open an issue to track this item?

Reply:
I don't have enough context for filing a good issue here. I'd recommend someone with more details of what is happening here to file the issue.